npm-shrinkwrap.json

```diff
@@ -2,11 +2,65 @@
   "name": "etcher",
   "version": "1.0.0-beta.18",
   "dependencies": {
+    "abbrev": {
```

Comment:
@jviotti I ran `make electron-develop`, `npm install --save angular-bind-html-compile-ci-dev` and `npm run shrinkwrap`.

`npm  --version` is `3.8.6`, `node --version` is `v6.1.0`

Am I missing something?